Tutorial: How to set up Flutter remote config (#10492)
PostHog/posthog.com: contents/tutorials/flutter-remote-config.md

            ),
            const SizedBox(height: 20),
            const Text(
              'Welcome message goes here', // We'll update this with remote config
              style: TextStyle(fontSize: 20),
            ),
          ],
        ),
      ),
    );
  }
}

```

To test this works, we check for the emulation devices available with `flutter devices` and run the app for one of them. We'll use the web version and launch it like this:

```bash
flutter run -d chrome
```

![App working](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2025_01_27_at_11_29_20_fc0b070bbb.png)

## 2. Add PostHog to your app

With our app set up, it's time to install and set up PostHog. If you don't have a PostHog instance, you can [sign up for free](https://us.posthog.com/signup).

To start, install [PostHog's Flutter SDK](/docs/libraries/flutter) by adding `posthog_flutter` to your `pubspec.yaml`:

```yaml
dependencies:
  flutter:
    sdk: flutter
  posthog_flutter: ^4.0.1
```

Next, we configure PostHog in each platform using our project API key and instance address which you can find in [your project settings](https://us.posthog.com/settings/project).

### Android setup

> For Android, add your PostHog configuration to your `AndroidManifest.xml` file located in the `android/app/src/main`:

Reviewer: ```suggestion
For Android, add your PostHog configuration to your `AndroidManifest.xml` file located at `android/app/src/main`:
```
